src/test/java/eu/yals/test/utils/mock/MattermostMock.java

```diff
@@ -6,81 +6,87 @@
 import java.util.ArrayList;
 
 /**
- * Build request string with comes from any Mattermost server
+ * Build request string with comes from any Mattermost server.
  *
  * @since 2.3
  */
 public class MattermostMock {
 
-  private final ArrayList<String> collector;
-  private final StringBuilder sb = new StringBuilder();
-
-  private MattermostMock() {
-    collector = new ArrayList<>(8);
-  }
-
-  public static MattermostMock create() {
-    return new MattermostMock();
-  }
-
-  public MattermostMock withChannelId(String channelId) {
-    collector.add(createParam(Marker.CHANNEL_ID, channelId));
-    return this;
-  }
-
-  public MattermostMock withChannelName(String channelName) {
-    collector.add(createParam(Marker.CHANNEL_NAME, channelName));
-    return this;
-  }
-
-  public MattermostMock withCommand(String command) {
-    collector.add(createParam(Marker.COMMAND, command));
-    return this;
-  }
-
-  public MattermostMock withTeamDomain(String teamDomain) {
-    collector.add(createParam(Marker.TEAM_DOMAIN, teamDomain));
-    return this;
-  }
-
-  public MattermostMock withTeamId(String teamId) {
-    collector.add(createParam(Marker.TEAM_ID, teamId));
-    return this;
-  }
-
-  public MattermostMock withText(String text) {
-    collector.add(createParam(Marker.TEXT, text));
-    return this;
-  }
-
-  public MattermostMock withToken(String token) {
-    collector.add(createParam(Marker.TOKEN, token));
-    return this;
-  }
-
-  public MattermostMock withUserId(String userId) {
-    collector.add(createParam(Marker.USER_ID, userId));
-    return this;
-  }
-
-  public MattermostMock withUsername(String username) {
-    collector.add(createParam(Marker.USER_NAME, username));
-    return this;
-  }
-
-  @Override
-  public String toString() {
-    sb.setLength(0);
-    for (String str : collector) {
-      sb.append(str).append(App.AND);
+    private final ArrayList<String> collector;
+    private final StringBuilder sb = new StringBuilder();
+
+    private MattermostMock() {
+        collector = new ArrayList<>(8);
+    }
+
+    public static MattermostMock create() {
+        return new MattermostMock();
+    }
+
+    /**
+     * Creates {@link MattermostMock} with given channel id.
+     *
+     * @param channelId string with channel id
+     * @return {@link MattermostMock}
+     */
+    public MattermostMock withChannelId(String channelId) {
+        collector.add(createParam(Marker.CHANNEL_ID, channelId));
+        return this;
+    }
+
+    public MattermostMock withChannelName(String channelName) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter channelName should be final.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)